salt/hg/config/legacy.apache.conf.jinja

```diff
@@ -116,4 +116,12 @@
     <Location /dev/peps>
         Header set X-Robots-Tag "noindex, nofollow"
     </Location>
+
+  <IfModule mod_ratelimit.c>
+    <Location /srv/hg/>
```

Comment:
I think the mercurial servers configured is in https://github.com/python/psf-salt/blob/main/salt/hg/config/hg.apache.conf.jinja

Its unclear if this configuration here will affect that server.